src/Avalonia.FuncUI.DSL/ToolTip.fs

```diff
@@ -0,0 +1,49 @@
+﻿namespace Avalonia.FuncUI.DSL
+
+[<AutoOpen>]
+module ToolTip =
+    open Avalonia.Controls
+    open Avalonia.Layout
+    open Avalonia.FuncUI.Types
+    open Avalonia.FuncUI.Builder
+   
+    let create (attrs: IAttr<ToolTip> list): IView<ToolTip> =
+        ViewBuilder.Create<ToolTip>(attrs)
+
+    type Control with
+
+        /// <summary>
+        /// A value indicating whether the tool tip is visible.
+        /// </summary>
+        static member isOpen<'t when 't :> Control>(value: bool) : IAttr<'t> =
+            AttrBuilder<'t>.CreateProperty<bool>(ToolTip.IsOpenProperty, value, ValueNone)
+       
+        /// <summary>
+        /// The content to be displayed in the control's tooltip.
+        /// </summary>
+        static member tip<'t when 't :> Control>(value: obj) : IAttr<'t> =
```

Comment:
ToolTips can contain other controls as a 'tip'. This is similar to ContentControls content property. see https://github.com/JaggerJo/Avalonia.FuncUI/blob/927b5dcd158d7cc9f7c742b04bfd355a9ff89521/src/Avalonia.FuncUI.DSL/ContentControl.fs#L12

Reply:
I added 3 more overloads as its in the ContentControl for the tip property